enterprise-suite/gotests/testenv/testenv.go

```diff
@@ -154,12 +155,11 @@ func cleanup(allowFailures bool) {
 		Expect(err).To(Succeed(), "lbc.Uninstall")
 	}
 
-	if foundStorageClass {
-		// Delete PVCs which are left after helm uninstall
-		for _, pvc := range consolePVCs {
-			if err := kube.DeletePvc(K8sClient, args.ConsoleNamespace, pvc); err != nil && !allowFailures {
-				Expect(err).To(Succeed(), "delete PVCs")
-			}
+	// Delete PVCs which are left after helm uninstall
+	for _, pvc := range consolePVCs {
+		if err := kube.DeletePvc(K8sClient, args.ConsoleNamespace, pvc); err != nil && !allowFailures {
+			Expect(err).To(Succeed(), "delete PVCs")
```

Comment:
Not sure this will work on local minishift. It doesn't have any storage classes, so tests will install `emptyDir` version of Console. Then here it will try to remove PVCs which were not created.

Reply:
It seemed to work without any PVCs in place. But I'll check again - it could also check it exists before deletion.

Reply:
Addressed it.